django-20-04/files/var/lib/digitalocean/settings.py

```diff
@@ -91,8 +91,9 @@ def ip_addresses():
         'NAME': 'django',
         'USER': 'django',
         'PASSWORD': '@DBPASSWORD@',
-        'HOST': 'localhost',
-        'PORT': '',
+        'HOST': '@DBHOST@',
+        'PORT': '@DBPORT@',
+        'OPTIONS': {'sslmode': 'require'},
```

Comment:
I believe this would break if using the local postgres instance, have we checked to make sure?

Reply:
Yep, I confirm local postgres instance works as intended and all migrations are successful
![image](https://user-images.githubusercontent.com/94634250/151149809-d14e936f-5a67-4264-9877-573774dc9163.png)